loom/script/native/core/system/lmConsole.cpp

```diff
@@ -36,8 +36,9 @@ int Console::print(lua_State *L)
     lua_rawgeti(L, 1, LSINDEXVECTOR);
     int vidx = lua_gettop(L);
 
-    // the value to print (concating all objects passed)
-    utString toprint;
+    // the value to print (concatenating all objects passed)
+    char outputBuffer[2048];
```

Comment:
Can you please add a trello card for this? I know alloca is sometimes finicky cross platform so it's worth exploring but out of scope for this PR.